### ISSUE TITLE
Is there a typo in the loss.py

### ISSUE THREAD

**Issue by qimiaohao** [varun19299/deep-atrous-guided-filter#4]
Hi：
  Very impressive work. I want to run the training code in my dataset. Is there a typo for the  line 58 in loss.py

self.total_loss += +self.image_loss + self.cobi_rgb_loss

what does it mean of “+=+”

is it:
self.total_loss = +self.image_loss + self.cobi_rgb_loss

Thank you very much

**Comment by varun19299** [comment]
`self.total_loss += +self.image_loss + self.cobi_rgb_loss` is syntactically correct, but should have been

`self.total_loss += self.image_loss + self.cobi_rgb_loss` (a typo).

Does this help?

**Comment by qimiaohao** [comment]
> `self.total_loss += +self.image_loss + self.cobi_rgb_loss` is syntactically correct, but should have been
> 
> `self.total_loss += self.image_loss + self.cobi_rgb_loss` (a typo).
> 
> Does this help?

Sorry. So it is correct? The correct code is still
 `self.total_loss += +self.image_loss + self.cobi_rgb_loss` ?
Thank you.

**Comment by varun19299** [comment]
We'll update it as `+= self.image_loss...`, but the existing version works too.

**Comment by qimiaohao** [comment]
Thank you very much
